assignments/assignment_5_md_version.md

```diff
@@ -0,0 +1,89 @@
+```python
+# This is just for you so that you can see where the numbers are coming from
+np.random.seed(12345)
+
+cars = np.random.randint(500,8000,size=500)
+np.save('./cars.npy',cars)
+
+value = np.random.uniform(size=500)
+np.save('./value.npy',value)
+```
+
+# Assignment 5
+
+Dan a university student wants to buy a car. He knows the price of each he is
+considering and he assigned a value per pound ratio to each of the cars (based
+on horsepower, consumption etc.). He saved this data in two arrays (clearly, why
+wouldn't he?), where the order of the cars was the same. Obviously Dan doesn't
+want a slow car, he wants the car to have at least $100$ horsepower, which start
+from $1500$ pounds. So Dan did the following:
+
+```python
+# Import numpy
+import numpy as np
+```
+
+First he loaded the arrays he created earlier, in the variable called cars and
+value using a numpy function
+[np.load()](https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.load.html).
+
+```python
+cars = np.load('./cars.npy')
```

Comment:
I guess they have to download these separately?

Reply:
Yes.